driver-core/src/main/com/mongodb/client/gridfs/model/GridFSDownloadByNameOptions.java

```diff
@@ -22,7 +22,9 @@
  * <p>Controls the selection of the revision to download</p>
  *
  * @since 3.1
+ * @deprecated use {@link GridFSDownloadOptions} instead.
  */
+@Deprecated
```

Comment:
I'd rather explicitly mark it for deprecation, but as the convention is not to mark code only used by deprecated methods I'll remove it.